rest/models/DashboardTemplate.go

```diff
@@ -48,15 +48,16 @@ const (
 	Ansible             AvailableWidgets = "ansible"
 	Rhel                AvailableWidgets = "rhel"
 	Openshift           AvailableWidgets = "openshift"
+	RecentlyVisited		AvailableWidgets = "recentlyVisited"
```

Comment:
@aferd can we fix the indentation of `AvailableWidgets` here?

Reply:
this is not showing on my side 😞 maybe if I fix it from here...